include/configcontainer.h

```diff
@@ -54,6 +70,7 @@ class configcontainer : public config_action_handler {
 	void reset_to_default(const std::string& key);
 	void toggle(const std::string& key);
 	std::vector<std::string> get_suggestions(const std::string& fragment);
+	SortStrategy get_current_sort_strategy();
```

Comment:
Since we have two sorting strategies—one for feeds, another for articles—I suggest to rename this method to `get_feed_sort_strategy` so that we don't have to do this later, when we add `get_article_sort_strategy`.

Speaking of which, do you want to add `get_article_sort_strategy`, or would you prefer me to open an issue and let someone deal with it later?

Reply:
>Speaking of which, do you want to add get_article_sort_strategy, or would you prefer me to open an issue and let someone deal with it later?

Oh, I forgot about article sort. I will add a method for articles soon (when I have a spare moment), so no need to open another ticket I think.